presto-verifier/src/main/java/com/facebook/presto/verifier/Validator.java

```diff
@@ -202,7 +208,7 @@ public boolean isDeterministic()
 
     private boolean validate()
     {
-        controlResult = executeQueryControl();
+        controlResult = executeQueryControl(null);
```

Comment:
Instead of passing null, you can change the method to accept `Optional<...>` type parameter and pass an `Optional.empty()`. Check the codebase for example :)